src/System.Runtime.Extensions/src/System/PlatformID.cs

```diff
@@ -2,16 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.ComponentModel;
+
 namespace System
 {
     public enum PlatformID
     {
-        Win32S = 0,
-        Win32Windows = 1,
+        [EditorBrowsable(EditorBrowsableState.Never)] Win32S = 0,
```

Comment:
It is most important to have these annotations in the reference assembly `src/System.Runtime.Extensions/ref/...`. The editor looks for the attributes in the reference assembly.

Annotating implementation is fine too.